v2src/flash/attn_bwd.cc

```diff
@@ -156,12 +156,12 @@ bwd_kernel_dk_dv(T4 q,
     .D = &delta,
     .num_head_q = num_head_q,
     .num_head_k = num_head_k,
+    .head_dim = head_size,
     .cu_seqlens_q = &cu_seqlens_q,
     .cu_seqlens_k = &cu_seqlens_k,
     .num_seqlens = num_seqlens,
     .max_seqlen_q = max_seqlen_q,
     .max_seqlen_k = max_seqlen_k,
-    .head_dim = head_size,
```

Comment:
Required because we're switching to gcc, which is more strict about the order of struct fields